### PR TITLE
fix configure_brightbox_repo in test

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by ekohl** [comment]
It appears this broke the tests. I do recall that some versions of the facts used 12.04 while others use 12.

**Comment by mmoll** [comment]
jep, it's depended on the version of facterdb as it seems, will have a look...

**Comment by mmoll** [comment]
simply checking for both strings now

File: spec/classes/foreman_install_spec.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@
             it { should contain_package('tfm-rubygem-passenger-native') }
           end
         when 'Debian'
-          configure_brightbox_repo = os == 'ubuntu-12-x86_64'
+          configure_brightbox_repo = os == 'ubuntu-12.04-x86_64' || os == 'ubuntu-12-x86_64'
```
Comment (ekohl) [inline spec/classes/foreman_install_spec.rb]: How about checking if the string starts with ```ubuntu-12```?

Comment (mmoll) [inline spec/classes/foreman_install_spec.rb]: that should also work :)

**Comment by domcleal** [comment]
Merged, thanks @mmoll.

**Pull request merged**